<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="5" rowspan="8">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_Connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QPushButton" name="pushButton_Disconnect">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="3">
     <widget class="QSlider" name="horizontalSlider_Min">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_Min"/>
    </item>
    <item row="4" column="0" rowspan="2" colspan="3">
     <widget class="QSlider" name="horizontalSlider_Max">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_Max"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Timing(s)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="3">
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_time">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonPut">
      <property name="text">
       <string>PutData</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="3">
     <widget class="QPushButton" name="pushButton_Stop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_Min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_Min</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_Max</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_time</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
